docs/sandbox/DataOverlay.md

```diff
@@ -0,0 +1,66 @@
+# Data Overlay
+TODO DataOverlay
+
+
+## Contact Info
+
+
+## Changelog
+- Initial version (December 2021)
+
+
+## Documentation
+TODO
+
+
+### Configuration 
+
+```json
+// otp-config.json
+{ "otpFeatures": { "DataOverlay" : true } }
+```
+
+```json
+// build-config.json
+{ "dataOverlay" : {
+  "fileName": "graphs/data-file.nc4",
```

Comment:
We discussed the "fileName" parameter today at the developer meeting. We think this is fine for now, but if someone want to access input data directly from the cloud, then we will have to change this. We will then most likely add a list of "data-overlay" data sources to the OTP storage. This will require the config to be changed, the `fileName` parameter will not be used by OTP any more - it will be ignored. Instead a list of data overlay data source should be added in the `storage` section in the _build-config.json_. We will also make the necessary changes to support multiple input files.